app/src/main/java/com/example/partycheckapp/presentation/feature/profile/view/ProfileFragment.kt

```diff
@@ -38,17 +89,70 @@ class ProfileFragment : MvpAppCompatFragment() {
 
         when (editFieldsMode) {
             true -> {
+                ib_edit.setImageResource(R.drawable.ic_save_black_24dp)
                 et_phone_num.keyListener = et_phone_num.tag as KeyListener?
                 et_card_num.keyListener = et_card_num.tag as KeyListener?
+                iv_photo.setOnClickListener { showDialog() }
             }
             false -> {
+                ib_edit.setImageResource(R.drawable.ic_mode_edit_black_24dp)
+                updateUser()
                 et_phone_num.keyListener = null
                 et_card_num.keyListener = null
+                iv_photo.setOnClickListener { }
                 this.context?.let { view?.let { it1 -> hideKeyboardFrom(it, it1) } }
             }
         }
     }
 
+    override fun showDialog() {
+        val ad = context?.let { AlertDialog.Builder(it) }
```

Comment:
context?.let { AlertDialog.Builder(it) }?.apply{
setTitle()
setPositive()
...
}